Update INL Application page

milljm opened the pull request:
Update our application page with links to internal resources. Fix peacock tests when PyQt isn't available (freebie).

Closes #22290

Commit:
Update INL Application page

Update our application page with links to internal resources.
Fix peacock tests when PyQt isn't available (freebie).

Closes #22290

Commit:
Forgot to update large_media

I am horrible at submodules.

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit d34af5ea41a54755f6cd5ef60150693f582b7f72
+Subproject commit 337c26d2b4eb642fe4cb0268bfed78037cf6133c
```
**Caution!** This contains a submodule update

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/1192137/) on fb54bc3 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/22297/site)

This comment will be updated on new commits.

milljm commented:
And documentation will keep failing until my large_media PR gets merged: https://github.com/idaholab/large_media/pull/32

joshuahansel commented on modules/doc/content/help/inl/applications.md:
```diff
@@ -10,10 +10,79 @@ breadth of applications continues to expand as well. The following sections high
+!row!
+!col small=12 medium=8 large=8
+[MARMOT Website](https://marmot-docs.hpcondemand.inl.gov/latest/)\\
+MARMOT is a mesoscale fuel performance code. As such, it can predict the evolution of the microstructure and material properties of fuels and claddings due to stress, temperature, and irradiation damage. MARMOT can, therefore, supply microstructure-based materials models to other code that works on engineering scale, which is larger than mesoscale, with an example being BISON. MARMOT solves equat ...
```
- to other applications that work at a larger scale, such as Bison, which works at an engineering scale.

joshuahansel commented on modules/doc/content/help/inl/applications.md:
```diff
@@ -10,10 +10,79 @@ breadth of applications continues to expand as well. The following sections high
+!col small=12 medium=4 large=4
+!row-end!
+
+## Relap-7
```
Capitalize RELAP

joshuahansel commented on modules/doc/content/help/inl/applications.md:
```diff
@@ -10,10 +10,79 @@ breadth of applications continues to expand as well. The following sections high
+!row!
+!col small=12 medium=8 large=8
+[Relap-7 Website](https://relap7-docs.hpcondemand.inl.gov/latest/)\\
+A next generation nuclear systems safety code, RELAP-7 takes advantage of advances in computer architecture, software design, numerical methods, and physical models for use in the Risk Informed Safety Margin Characterization (RISMC) methodology and in nuclear power plant (NPP) safety analysis. RELAP-7 is a more capable than its predecessors in the RELAP family due to better flow models, improved  ...
```
Replace the whole description. I suggest:

RELAP-7 is a two-phase thermal systems code based on MOOSE's thermal hydraulics module. RELAP-7 provides two-phase components suitable for LWRs provides closures for two-phase water from TRACE.

joshuahansel commented on modules/doc/content/help/inl/applications.md:
```diff
@@ -10,10 +10,79 @@ breadth of applications continues to expand as well. The following sections high
+!row!
+!col small=12 medium=8 large=8
+[Sockeye Website](https://sockeye-docs.hpcondemand.inl.gov/latest/)\\
+Sockeye is a MOOSE-based heat pipe simulator and analysis tool. It, therefore, provides the ability to accurately predict heat transfer for heat-pipe-cooled microreactors and other heat pipe applications. Importantly, Sockeye models heat conduction transients in 1D and 2D as well as offering tools to analyze the operating envelope of heat pipes. So, it provides insight into operational limits in  ...
```
Replace the whole description. I suggest:

Sockeye is a heat pipe analysis application geared towards heat pipes of interest in heat-pipe-cooled microreactors. Sockeye provides various transient heat pipe modeling capabilities, including a 1D, two-phase flow model and a 2D effective thermal conductivity model based on heat conduction, which leverages analytic expressions of operational limits.

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit d34af5ea41a54755f6cd5ef60150693f582b7f72
+Subproject commit cf30a5de0cdbfe33502b0d4fb01cee09b9f8391d
```
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

Commit:
Corrected filenames, adjusted descriptions

Addressed change requests to better describe the application.
Corrected filenames.
